src/features/TokenMigration/TokenMigrationStepper.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect } from 'react';
+
+import { useStateReducer, useTxMulti } from '@utils';
+import { ITxReceipt, ITxConfig, TxParcel, ITxStatus } from '@types';
+import { default as GeneralStepper, IStepperPath } from '@components/GeneralStepper';
+import { ROUTE_PATHS } from '@config';
+import { translateRaw } from '@translations';
+import { WALLET_STEPS } from '@components';
+
+import { tokenMigrationConfig } from './config';
+import TokenMigrationInteractionFactory from './stateFactory';
+import { ISimpleTxFormFull, TokenMigrationState, ITokenMigrationFormFull } from './types';
+import { createMigrationTx, createApproveTx } from './helpers';
+import { isERC20Tx } from '../SendAssets';
+import TokenMigrationForm from './components/TokenMigrationForm';
+import { TokenMigrationMultiTx } from './components';
+import TokenMigrationReceipt from './components/TokenMigrationReceipt';
+
+const initialTokenMigrationFlowState = {
+  tokenConfig: tokenMigrationConfig,
+  txConfig: (undefined as unknown) as ITxConfig,
+  txReceipt: (undefined as unknown) as ITxReceipt
+};
+
+const TokenMigrationStepper = () => {
```

Comment:
E2E test

Reply:
Puttng this off till we extract Token Migration to a more abstracted interaction.